package.json

```diff
@@ -6,7 +6,7 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
     "dev": "ALGOLIA_APP_ID='.' ALGOLIA_SEARCH_API_KEY='.' docusaurus start",
-    "build": "docusaurus build",
+    "build": "NODE_OPTIONS=--openssl-legacy-provider && docusaurus build",
```

Comment:
Is `NODE_OPTIONS=--openssl-legacy-provider` a workaround to handle deprecations or breaking changes in newer versions of Node.js related to OpenSSL?

Reply:
Actually, after doing some more research: this should not be required. Need @MartinMinkov to check if Vercel uses the latest LTS v18 to build the project, if so, I can revert that workaround